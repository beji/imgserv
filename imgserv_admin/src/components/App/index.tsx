import React from "react";
import { Link } from "react-router-dom";

class App extends React.Component {
    public render() {
        return (
            <React.Fragment>
                {this.props.children}
            </React.Fragment>
        );
    }
}

export default App;
