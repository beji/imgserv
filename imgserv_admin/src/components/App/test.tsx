import React from "react";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import App from ".";

test("App is rendered correctly", () => {
    const component = renderer.create(<BrowserRouter><App /></BrowserRouter>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
