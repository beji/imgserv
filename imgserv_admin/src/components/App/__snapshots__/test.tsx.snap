// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App is rendered correctly 1`] = `
<header
  className="sc-bdVaJa GoLQP"
>
  <a
    href="/"
    onClick={[Function]}
  >
    Header
  </a>
</header>
`;
